example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -405,8 +407,14 @@ describe('When there are granule differences and granule reconciliation is run',
         granuleId: publishedGranuleId,
         collectionId,
       });
-      console.log('XXXXX Completed getGranule()');
-      await waitForGranuleRecordUpdatedInList(config.stackName, granuleBeforeUpdate);
+      console.log('XXXXX Completed for getGranule()');
+      await waitForGranuleRecordUpdatedInList(
+        config.stackName,
+        granuleBeforeUpdate,
+        {
+          includeFullRecord: 'true',
```

Comment:
Nit: this could probably be condensed into a DRYed up param to avoid all the bracket extra lines.

Reply:
good point
https://github.com/nasa/cumulus/pull/3853/commits/bb18a572f64ced69253a94d8728d758c94741195